base/src/main/java/com/maubis/scarlet/base/main/specs/MainActivityBottomBarSpec.kt

```diff
@@ -52,14 +53,17 @@ object MainActivityBottomBarSpec {
         })
     row.child(EmptySpec.create(context).heightDip(1f).flexGrow(1f))
 
-    row.child(bottomBarRoundIcon(context, colorConfig)
-        .iconRes(R.drawable.icon_add_notebook)
-        .onClick {
-          CreateOrEditFolderBottomSheet.openSheet(
+    if (!isInsideFolder) {
+      row.child(bottomBarRoundIcon(context, colorConfig)
+          .iconRes(R.drawable.icon_add_notebook)
+          .onClick {
+            CreateOrEditFolderBottomSheet.openSheet(
```

Comment:
Actually instead of hiding the folder icon which will be jarring for the user, can you instead do this:
```
row.child(bottomBarRoundIcon(context, colorConfig)
....
.alpha(if (isInsideFolder) 0.4f else 1.0f)
.onClick {
  if (isInsideFolder) {
    return@onClick
  }
}

```